.github/workflows/deploy.yml

```diff
@@ -12,21 +11,30 @@ jobs:
         id: branch_check
         run: |
           echo "Running on branch ${{ github.ref }}"
-          if [ "${{ github.ref }}" = "refs/heads/master" ]; then
-            echo "::set-output name=env_name:production"
-          elif [ "${{ github.ref }}" = "refs/heads/main" ]; then
-            echo "::set-output name=env_name:production"
+          if [ "${{ github.ref }}" = "refs/heads/master" ] || [ "${{ github.ref }}" = "refs/heads/main" ]; then
```

Comment:
@Pralish this looks good, can you update [the wiki](https://github.com/restarone/violet_rails/wiki/Blue-Green-deployment-using-a-Github-Action) with the new changes?